services-turf/src/main/java/com/mapbox/turf/TurfMeta.java

```diff
@@ -142,6 +145,76 @@ private TurfMeta() {
     return coords;
   }
 
+  /**
+   * Get all coordinates from a {@link Feature} object, returning a {@link List} of {@link Point}
+   * objects.
+   *
+   * @param feature the {@link Feature} that you'd like to extract the Points from.
+   * @return a {@code List} made up of {@link Point}s
+   * @since 4.7.0
+   */
+  public static List<Point> coordAll(@NonNull Feature feature) {
+    return coordAll(FeatureCollection.fromFeature(feature));
+  }
+
+  /**
+   * Get all coordinates from a {@link FeatureCollection} object, returning a
+   * {@link List} of {@link Point} objects.
+   *
+   * @param featureCollection the {@link FeatureCollection} that you'd like
+   *                          to extract the Points from.
+   * @return a {@code List} made up of {@link Point}s
+   * @since 4.7.0
+   */
+  public static List<Point> coordAll(@NonNull FeatureCollection featureCollection) {
+    List<Point> finalCoordsList = new ArrayList<>();
+    for (Feature singleFeature : featureCollection.features()) {
+      finalCoordsList.addAll(coordAll(singleFeature.geometry()));
```

Comment:
This will result in extra Lists creates (one per feature).
We could have a helper method that would add to existing list
```
private static List<Point>  addCoordAll(@NonNull List<Point> pointList, @NonNull Feature feature)
```
and
```
public static List<Point> coordAll(@NonNull Feature feature) {
    return addCoordAll(new ArrayList<Point>, @NonNull Feature feature);
}
```